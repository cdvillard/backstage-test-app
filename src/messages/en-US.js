export default {
  //App
  'app.page.title': '{app}',

  // Error Boundary
  'errorBoundary.app.title': 'Uh oh. {app} failed to load properly.',
  'errorBoundary.app.desc': 'There was an error. Please refresh the page. You may need to clear caches and reopen your browser. If the issue persists, contact support.',
  'errorBoundary.app.refresh': 'Refresh',
  'errorBoundary.app.support': 'Contact Support',
  'errorBoundary.app.default.name': 'This app',

  // Not Found
  'notFound': 'Not Found',
  'notFound.error404': '404: Page Not Found',
  'notFound.title': 'Oops, looks like this link didn\'t work!',
  'notFound.description': 'It seems that you linked to a page that doesn\'t exist.',
  'notFound.button': 'Take me to the dashboard',
  'notFound.explanations.title': 'Why am I seeing this?',
  'notFound.explanationOne.title': 'Outdated bookmark',
  'notFound.explanationOne.body': 'We moved a page that you have bookmarked. Please remove or update the bookmark.',
  'notFound.explanationTwo.title': 'Page doesn\'t exist',
  'notFound.explanationTwo.body': 'You tried to access a page that doesn\'t exist.  Please double check your link.',
  'notFound.explanationThree.title': 'Page moved',
  'notFound.explanationThree.body': 'You clicked a link to a page that was moved.',

  // No Permissions
  'noPermissions': 'No Permissions',
  'noPermissions.description': 'You don\'t have permissions to see resources in this space.',
  'noPermissions.nextSteps': 'Go to {dashboardLink} and select desired tenant and space in the navigation pane.',
  'noPermissions.dashboard': 'Dashboard',
  'noPermissions.requestAccess': 'Request Access',
  'noPermissions.documentation': 'Documentation',

  // Refresh Prompt
  'refresh.prompt.title': 'Your session will expire soon',
  'refresh.prompt.text': 'Refresh your browser to extend your session, otherwise you will be logged out.',
  'refresh.prompt.continue': 'Refresh',
  'refresh.prompt.logout': 'Log Out',
};
