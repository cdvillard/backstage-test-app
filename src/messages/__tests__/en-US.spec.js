 import messages from '../en-US';

describe('en-US messages', () => {
  it('creates snapshot', () => {
    expect(messages).toMatchSnapshot();
  });
});
