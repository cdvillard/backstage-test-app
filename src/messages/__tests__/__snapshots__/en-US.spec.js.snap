// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`en-US messages creates snapshot 1`] = `
Object {
  "errorBoundary.app.default.name": "This app",
  "errorBoundary.app.desc": "There was an error. Please refresh the page. You may need to clear caches and reopen your browser. If the issue persists, contact support.",
  "errorBoundary.app.refresh": "Refresh",
  "errorBoundary.app.support": "Contact Support",
  "errorBoundary.app.title": "Uh oh. {app} failed to load properly.",
  "noPermissions": "No Permissions",
  "noPermissions.dashboard": "Dashboard",
  "noPermissions.description": "You don't have permissions to see resources in this space.",
  "noPermissions.documentation": "Documentation",
  "noPermissions.nextSteps": "Go to {dashboardLink} and select desired tenant and space in the navigation pane.",
  "noPermissions.requestAccess": "Request Access",
  "notFound": "Not Found",
  "notFound.button": "Take me to the dashboard",
  "notFound.description": "It seems that you linked to a page that doesn't exist.",
  "notFound.error404": "404: Page Not Found",
  "notFound.explanationOne.body": "We moved a page that you have bookmarked. Please remove or update the bookmark.",
  "notFound.explanationOne.title": "Outdated bookmark",
  "notFound.explanationThree.body": "You clicked a link to a page that was moved.",
  "notFound.explanationThree.title": "Page moved",
  "notFound.explanationTwo.body": "You tried to access a page that doesn't exist.  Please double check your link.",
  "notFound.explanationTwo.title": "Page doesn't exist",
  "notFound.explanations.title": "Why am I seeing this?",
  "notFound.title": "Oops, looks like this link didn't work!",
  "refresh.prompt.continue": "Refresh",
  "refresh.prompt.logout": "Log Out",
  "refresh.prompt.text": "Refresh your browser to extend your session, otherwise you will be logged out.",
  "refresh.prompt.title": "Your session will expire soon",
}
`;
